dashboard/app/models/school.rb

```diff
@@ -332,7 +345,7 @@ def load_state_cs_offerings(offerings_to_load, is_dry_run)
   # @param filename [String] The CSV file name.
   # @param options [Hash] The CSV file parsing options.
   # @param write_updates [Boolean] Specify whether existing rows should be updated.  Default to true for backwards compatible with existing logic that calls this method to UPSERT schools.
-  def self.merge_from_csv(filename, options = CSV_IMPORT_OPTIONS, write_updates = true, is_dry_run: false, new_attributes: [])
+  def self.merge_from_csv(filename, options = CSV_IMPORT_OPTIONS, write_updates = true, is_dry_run: false, new_attributes: [], write_inserts: true)
```

Comment:
Added this `write_inserts` check to prevent attempting to write new rows for a small number of schools in this dataset that aren't in the schools table already. Trying to create a row with just latitude/longitude fails, as a school name is required.

Reply:
I'm curious, why wouldn't we want to add new schools to the table?

Reply:
This PR is supposed to just "fill" latitude and longitude for schools that will be uploaded via https://github.com/code-dot-org/code-dot-org/pull/38432, rather than introduce new schools. Schools are required to have a name, so any attempt to add a school from this data set fails. 

In an ideal world, there shouldn't be any rows in this dataset that aren't already uploaded as part of the schools upload PR, but our schools data is a bit of a wild west and there are always a few rows that are skipped for various reasons. That said, I'm confident it's very few rows that fall into this bucket b/c the sum of the # of schools updated and unchanged in the output (~102K) is close to the number of rows in the file.